src/xmipp/libraries/reconstruction/resolution_pdb_bfactor.cpp

```diff
@@ -325,22 +325,22 @@ void ProgResBFactor::generateOutputPDB(const std::vector<double> &residuesToChim
 
 		if ( (typeOfline == "ATOM") || (typeOfline == "HETA"))
 		{
-			std::string lineInit = line.substr(0,61);
+			std::string lineInit = line.substr(0,60);
 			std::string lineEnd = line.substr(66,  std::string::npos);
 
-                        // The residue is read
+			// The residue is read
 			auto resi = (int) textToFloat(line.substr(23,5));
 			std::string lineMiddle;
 			int digitNumber = 5;
 			
 			std::string auxstr = std::to_string(residuesToChimera[resi-1]);
 
-			// The bfactor column has 5 digits so we set the normalized resolution to 
-            // 5 digits
+			// The bfactor column has 6 characters so we set the normalized resolution to 
+			// 5 characters with an empty character at the front
 			auxstr = auxstr.substr(0, 5);
-                        std::stringstream ss;
-			ss << std::setfill('0') << std::setw(5) << residuesToChimera[resi-1];
-                        lineMiddle = ss.str();
+			std::stringstream ss;
+			ss << std::setfill(' ') << std::setw(6) << auxstr;
```

Comment:
Just to know... Why do you want to set with a white space instead of with 0? I do not know the behaviour when chimera will open the pdb

Reply:
We were filling the next column with a zero. If we fill up to this many columns with 0 then that doesn't make sense e.g. we'd have 00.1

Most lines in most pdb files have a white space in the column I'm now pointing to.

Reply:
Ohh I see, thanks